### PR TITLE
Correctly uses tidb cluster version var when creating cluster

### PR DESCRIPTION
**Pull request by jlerche**
### What problem does this PR solve? 
GCP terraform was not creating tidb clusters when `tidb_version` was overriden with the right version number

### What is changed and how does it work?
Correctly passes tidb version through the modules

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

 - Has terraform changes

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:
 
 ```release-note
NONE
 ```

### CODE REVIEW

File: deploy/modules/gcp/tidb-cluster/variables.tf [hunk 1]
```diff
@@ -47,15 +47,15 @@ variable "monitor_instance_type" {
 
 variable "pd_image_type" {
   description = "PD image type, avaiable: UBUNTU/COS"
-  default = "COS"
+  default     = "COS"
```
Comment (gregwebs) [inline deploy/modules/gcp/tidb-cluster/variables.tf]: this will merge conflict with existing PRs

**Comment by jlerche** [comment]
@cofyc PTAL to make sure this does not conflict with your PRs.

**Pull request merged**